Testscripts/Linux/utils.sh

```diff
@@ -1368,7 +1368,6 @@ CreateIfupConfigFile()
 						BOOTPROTO=none
 						IPADDR="$__ip"
 						NETMASK="$__netmask"
-						NM_CONTROLLED=no
```

Comment:
what is the usage of this setting? does this change will affect other version of RHEL? like RHEL6.X?

Reply:
No, will not affect other version of RHEL.

Reply:
The first fix has some issue, will pull request later after fix, sorry for the noise.